src/Sylius/Behat/Context/Api/Admin/ManagingOrdersContext.php

```diff
@@ -311,4 +315,34 @@ public function theAdministratorShouldSeeThatThisOrderHasBeenPlacedIn(
 
         Assert::same($currencyCode, $currency);
     }
+
+    /**
+     * @Then I should see an order with :orderNumber number
+     */
+    public function iShouldSeeOrderWithNumber($orderNumber)
+    {
+        $order = $this->orderRepository->findOneBy(['number' => $orderNumber]);
+
+        Assert::notNull($order, sprintf('Cannot find order with number %s', $orderNumber));
+    }
+
+    /**
+     * @When I switch the way orders are sorted by :fieldName
+     */
+    public function iSwitchSortingBy($fieldName)
+    {
+      $this->client->addFilter('order[number]', 'asc');
+    }
+
+    /**
+     * @Then the first order should have number :number
+     */
+    public function theFirstOrderShouldHaveNumber($number)
+    {
+        $this->client->filter();
```

Comment:
I would move it to the `When` step right after `$this->client->addFilter(/*...*/)`